src/renderer/components/+cluster/cluster-no-metrics.tsx

```diff
@@ -3,20 +3,33 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
+import styles from "./cluster-no-metrics.module.scss";
+
 import React from "react";
 import { Icon } from "../icon";
 import { cssNames } from "../../utils";
+import { broadcastMessage } from "../../../common/ipc";
+import { IpcRendererNavigationEvents } from "../../navigation/events";
+import { catalogEntityRegistry } from "../../api/catalog-entity-registry";
 
 export interface ClusterNoMetricsProps {
   className: string;
 }
 
 export function ClusterNoMetrics({ className }: ClusterNoMetricsProps) {
+  function getClusterId() {
+    return catalogEntityRegistry.activeEntity;
+  }
+
+  function openSettingsPage() {
+    broadcastMessage(IpcRendererNavigationEvents.NAVIGATE_IN_APP, `/entity/${getClusterId()?.getId()}/settings`);
```

Comment:
Could this be a dependency? @Iku-turso does your work currently support this sort of navigation?

Reply:
If this is a dependency, then we could have a unit test as well as the e2e test.

Reply:
If `getClusterId()` will be `undefined` would it be still a valid route? Maybe we should not send anything if cluster not exists?